Makefile

```diff
@@ -27,6 +27,9 @@ nginx-build-loris: install-docker-build-deps
 nginx-build-services: install-docker-build-deps
 	./scripts/build_nginx_image.py --variant=services
 
+nginx-build-grafana: install-docker-build-deps
```

Comment:
You need to add the new tasks to the catchall `nginx-build` and `nginx-deploy` tasks. Then new images will be built and pushed by Travis. \o/